.github/workflows/ci-build-image.yml

```diff
@@ -0,0 +1,79 @@
+name: CRDA Build and Push Image 
+on:
+  issue_comment:
+    types: [created]
+
+env:
+  REGISTRY:  ghcr.io
+  REPOSITORY: ${{ github.repository }}
+  DEFAULT_TAG: ${{ github.event.issue.number }}
+  FULL_NAME: bayesian-bayesian-api
+ 
+
+jobs:
+  Build-push-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git fetch origin +refs/pull/${{ github.event.issue.number }}/merge
+          git checkout FETCH_HEAD
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
+        id: check
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commands: |
+            build-image
+      - name: send comment 
+        env:
+          URL: ${{ github.event.issue.comments_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl \
```

Comment:
used GITHUB's API because there is no action which would let me post a comment on the PR when the trigger was an issue comment